tests/unit/model.py

```diff
@@ -44,16 +44,19 @@ def configure():
 class TestNic(ModelTest):
 
     def sample_obj(self):
-        return Nic(Node(label='node-99', obm=Ipmi(type="http://schema.massopencloud.org/haas/v0/obm/ipmi",
-                  host= "ipmihost", user= "root", password= "tapeworm")), 'ipmi', '00:11:22:33:44:55')
-        
+        return Nic(Node(label='node-99',
+                        obm=Ipmi(type="http://schema.massopencloud.org/haas/v0/obm/ipmi",  # noqa
```

Comment:
This one is  a bit tought to deal with. Could we do `Ipmi.ap_name` instead? Has some other advantages as  well.

For that matter, a reasonable future cleanup task might be to add an `__init__` to these that supplies the type, so you can just omit it. *that* is probably out of scope though.

Do others have thoughts on just using `api_name`?